backend/tests/unit/records/marshmallow/literature/common/test_reference_item.py

```diff
@@ -241,3 +241,39 @@ def test_returns_arxiv_eprints_from_the_resolved_record(
     expected = {"control_number": 123, "arxiv_eprint": [{"value": "1606.09129"}]}
     result = schema.dumps(dump).data
     assert expected == orjson.loads(result)
+
+
+@mock.patch("inspirehep.records.api.base.InspireRecord.get_records_by_pids")
+def test_doesnt_raise_key_error_missing_fullname(get_records_mock):
+    schema = ReferenceItemSchemaV1()
+    dump = {
+        "reference": {
+            "label": "123",
+            "authors": [],
+            "publication_info": {"journal_title": "Alias Investigations"},
+            "title": {"title": "Jessica Jones"},
+            "arxiv_eprint": "1207.7214",
+            "dois": ["10.1016/j.physletb.2012.08.020"],
+            "urls": [
+                {"value": "http://www.claymath.org/prize-problems"},
+                {"value": "http://www.arthurjaffe.com"},
+            ],
+            "collaborations": ["CMS", "ATLAS Team"],
+        }
+    }
+    expected = {
+        "label": "123",
+        "publication_info": [{"journal_title": "Alias Investigations"}],
+        "titles": [{"title": "Jessica Jones"}],
+        "arxiv_eprint": [{"value": "1207.7214"}],
+        "dois": [{"value": "10.1016/j.physletb.2012.08.020"}],
+        "urls": [
+            {"value": "http://www.claymath.org/prize-problems"},
+            {"value": "http://www.arthurjaffe.com"},
+        ],
+        "collaborations": [{"value": "CMS"}],
+        "collaborations_with_suffix": [{"value": "ATLAS Team"}],
+    }
+    record = LiteratureRecord(dump)
+    result = schema.dumps(record).data
+    assert expected == orjson.loads(result)
```

Comment:
hmm I'm very confused, this test is not related to the code changes, you changed something in the CVSchema :/ Do I miss something?

Reply:
Thank you for pointing that out. I don't know how I managed to get so confused with the codes that I wrote the test in the wrong place.